content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+  - जब कोई पीआर किसी मौजूदा पेज को अपडेट करता है, तो आपको पेज के उन हिस्सों की समीक्षा करने पर ध्यान देना चाहिए जिन्हें अपडेट किया जा रहा है। उस बदली हुई सामग्री की तकनीकी और संपादकीय शुद्धता के लिए समीक्षा की जानी चाहिए।
+    यदि आपको पृष्ठ पर त्रुटियां मिलती हैं जो पीआर लेखक द्वारा संबोधित करने का प्रयास करने से सीधे संबंधित नहीं होती हैं, तो इसे एक अलग मुद्दे के रूप में माना जाना चाहिए (पहले जांचें कि इस बारे में कोई मौजूदा समस्या तो नहीं है)।
+  - सामग्री को _स्थानांतरित_ करने वाले पुल अनुरोधों से सावधान रहें। यदि कोई लेखक किसी पृष्ठ का नाम बदलता है या दो पृष्ठों को जोड़ता है, तो हम (कुबेरनेट्स SIG Docs) आमतौर पर उस लेखक से प्रत्येक व्याकरण या वर्तनी इकाई को ठीक करने के लिए कहने से बचते हैं जिसे हम उस स्थानांतरित सामग्री में खोज सकते हैं।
+- क्या कोई जटिल या पुराना शब्द है जिसे सरल शब्द से बदला जा सकता है?
+- क्या ऐसे कोई शब्द या वाक्यांश उपयोग में हैं जिन्हें गैर-भेदभावपूर्ण विकल्प से बदला जा सकता है?
+- क्या शब्द का चुनाव और उसका कैपिटलाइज़ेशन [शैली गाइड](/docs/contribute/style/style-guide/) का पालन करता है?
+- क्या ऐसे लंबे वाक्य हैं जो छोटे या कम जटिल हो सकते हैं?
+- क्या कोई लंबा पैराग्राफ है जो सूची या तालिका के रूप में बेहतर काम कर सकता है?
+
+### सामग्री
+
+- क्या इसी तरह की सामग्री कुबेरनेट्स साइट पर कहीं और मौजूद है?
+- क्या सामग्री ऑफ-साइट, व्यक्तिगत विक्रेता या गैर-ओपन सोर्स दस्तावेज़ीकरण से अत्यधिक लिंक करती है?
+
+### वेबसाइट
+
+- क्या इस पीआर ने पृष्ठ शीर्षक, स्लग/उपनाम या एंकर लिंक को बदल दिया या हटा दिया? यदि हां, तो क्या इस पीआर के परिणामस्वरूप टूटे लिंक हैं? क्या कोई अन्य विकल्प है, जैसे स्लग को बदले बिना पृष्ठ का शीर्षक बदलना?
+
+- क्या पीआर एक नया पृष्ठ पेश करता है? यदि ऐसा है तो:
+  - क्या पृष्ठ सही [पृष्ठ सामग्री प्रकार](/docs/contribute/style/page-content-types/) और संबद्ध ह्यूगो शॉर्टकोड का उपयोग कर रहा है?
+  - क्या पेज सेक्शन के साइड नेविगेशन में ठीक से दिखाई देता है?
+  - क्या पेज को [डॉक्स होम](/hi/docs/home/) लिस्टिंग में दिखना चाहिए?
+
+- क्या परिवर्तन Netlify पूर्वावलोकन में दिखाई देते हैं? सूचियों, कोड, ब्लॉकों, तालिकाओं, नोट्स और छवियों के बारे में विशेष रूप से सतर्क रहें।
+
+### अन्य
+
+- इसे ध्यान से देखें [सरल संपादन](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits);
+  यदि आपको कोई परिवर्तन दिखाई देता है जो आपको लगता है कि एक सरल संपादन है, तो कृपया उस नीति को इंगित करें
+  (यदि यह वास्तव में सुधार है तो परिवर्तन को स्वीकार करना अभी भी ठीक है)।
+- उन लेखकों को प्रोत्साहित करें जो अपने पीआर की पहली प्रतिबद्धता में ऐसा करने के लिए व्हॉट्सएप फिक्स कर रहे हैं, और उसके बाद अन्य परिवर्तन जोड़ें। यह विलय और समीक्षा दोनों को आसान बनाता है। विशेष रूप से एक छोटे से परिवर्तन के लिए देखें जो एक बड़ी मात्रा में व्हाइटस्पेस क्लीनअप के साथ होता है (और यदि आप इसे देखते हैं, तो लेखक को इसे ठीक करने के लिए प्रोत्साहित करें)।
+
+एक समीक्षक के रूप में, यदि आप पीआर के साथ छोटे मुद्दों की पहचान करते हैं जो अर्थ के लिए आवश्यक नहीं हैं, जैसे टाइपो या गलत व्हाइटस्पेस, अपनी टिप्पणियों को `nit:` के साथ उपसर्ग करें।
```

Comment:
```suggestion
एक रिव्युवर के रूप में, यदि आप PR के साथ छोटे मुद्दों की पहचान करते हैं, जो अर्थ के लिए आवश्यक नहीं हैं, जैसे टाइपो या गलत व्हाइटस्पेस, अपनी टिप्पणियों को `nit:` के साथ उपसर्ग करें।
```

Reply:
I would stick with समीक्षक and instead of पीआर / PR, I would recommend पुल रिक्वेस्ट